src/components/CoreApps/index.js

```diff
@@ -0,0 +1,88 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import React from 'react'
+import getLatestVersion from '../../get-latest-version'
+import AppList from '../AppList'
+
+const coreAppNames = [
+    'App Management',
+    'Cache Cleaner',
+    'Capture',
+    'Dashboard',
+    'Data Visualizer',
+    'Datastore Management',
+    'Import/Export',
+    'Interpretations',
+    'Maintenance',
+    'Menu Management',
+    'Messaging',
+    'Reports',
+    'SMS Configuration',
+    'Scheduler',
+    'Settings',
+    'Translations',
+    'User Management',
+]
```

Comment:
These should be, again, short names rather than names.  They should then be prefixed with `dhis-web` to get the namespace of that application (i.e. `dhis-web-app-management`, `dhis-web-capture`, etc - look in the URL when visiting an app to see this).

This is also missing several bundled / core applications, notably the STRUTS apps - the full list is [here](https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-api/src/main/java/org/hisp/dhis/appmanager/AppManager.java#L49-L81)

Finally - when this switches to using `short_name` we should be fetching the translations of bundled applications from the Core (POST /api/translations or displayName from GET /dhis-web-commons/menu/getModules.action), rather than always showing them in English) - this could be a future enhancement if it's too much work for now as no other apps are currently translated, but would be a nice-to-have.